lib/matplotlib/tests/test_contour.py

```diff
@@ -375,3 +375,17 @@ def test_contour_uneven():
     ax = axs[1]
     cs = ax.contourf(z, levels=[2, 4, 6, 10, 20])
     fig.colorbar(cs, ax=ax, spacing='uniform')
+
+
+@pytest.mark.parametrize("rcll, rccl, fcl, expected", [
+    (1.23, None, None, 1.23),
+    (1.23, 4.24, None, 4.24),
+    (1.23, 4.24, 5.02, 5.02)
+])
+def test_contour_linewidths(rcll, rccl, fcl, expected):
```

Comment:
Please use more descriptive parameter names. Finding out what these mean is unnecessarily difficult.